packages/react-storage/jest.config.ts

```diff
@@ -16,9 +16,9 @@ const config: Config = {
       // functions: 90,
       // lines: 95,
       // statements: 95,
-      branches: 76,
+      branches: 74,
       functions: 77,
-      lines: 87,
+      lines: 86,
```

Comment:
Would prefer not to lower unit test thresholds at this point unless absolutely required

Reply:
makes sense. When I was running the tests my coverage was not meeting the threshold even though I got most of the coverage for the files I added. I just merged in the latest changes and was able to remove the code for `LocationDetailViewTable`

After this, the tests are now passing at the original threshold.

Reply:
@timngyn If the tests are passing at the original threshold, can we remove this change?

Reply:
@reesscot yea, I reverted them, but didn't push up my changes yet until just now